account_invoice_subscription_per_contact/models/account_move.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2025 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        res = super().create(vals_list)
+        invoice_create_subtype = self.env.ref(
+            "account_invoice_subscription_per_contact.mt_invoice_new"
+        )
+        for item in res:
+            partner = item.partner_id.commercial_partner_id
+            partner_ids = partner.message_follower_ids.filtered(
+                lambda x: invoice_create_subtype in x.subtype_ids
+            ).mapped("partner_id")
+            if partner_ids:
+                item.message_subscribe(partner_ids=partner_ids.ids)
```

Comment:
You are not caring here about which subtypes are subscribed, but anyway, see my previous comment. I think we should put the standard ones (like this) + "Invoice Created" subtype in the move.